setup.py

```diff
@@ -37,7 +38,7 @@
     install_requires=[
         'attrs',
         'funcsigs',
-        'requests-mock',
+        'requests-mock<2.0.0',
```

Comment:
:/

Reply:
If not needed may be we should not freeze this main requirement. Let the user do that if he need it. Should be ok 'til we maintain it well ;) (thx to you @mrjmad & @joehybird )